tools/build/configure.py

```diff
@@ -123,6 +123,11 @@ def write_ninja_rules(ninja: ninja_syntax.Writer, cpp: str, cppflags: str, extra
         deps="gcc",
     )
 
+    ninja.rule("dead_cc",
+        description="dead_cc $in",
+        command=f"mips-linux-gnu-objcopy --redefine-sym sqrtf=dead_sqrtf $in $out",
```

Comment:
Can we call this rule `dead_objcopy` or something? It isn't cc at least

Reply:
Perhaps instead of dead.h we should do `--redefine-sym` for all the dead syms here in the future :thinking: